test/dynamo/test_minifier.py

```diff
@@ -326,54 +253,6 @@ def test_dynamo_config_serialization(self):
         proc, _ = self._run_test_code(run_code)
         self.assertEqual(proc.returncode, 0)
 
-    # Test that launched minifier processes have the same config as
-    # the original process.
-    def _test_after_dynamo_with_modified_config(
```

Comment:
This test is now pointless; if config is not propagated, the bugs won't propagate (as they are propagated by config!)